tsconfig.json

```diff
@@ -1,18 +1,20 @@
 {
-    "compilerOptions": {
-      "target": "esnext",
-      "module": "esnext",
-      "moduleResolution": "node",
-      "skipLibCheck": true,
-  
-      "allowJs": true,
-      "checkJs": true,
-      "strict": true,
-  
-      "noEmit": true,
-    },
-    "include": [
-      ".eslintrc.cjs",
-    ]
-  }
-  
\ No newline at end of file
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+
+    "allowJs": true,
+    "checkJs": true,
+    "strict": false,
+    "downlevelIteration": true,
```

Comment:
I still think this should be removed, because it's irrelevant with `noEmit: true` but for the same reason it's not a big deal.